app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@ class UsersController < ApplicationController
   before_action :set_user, only: [:update, :questions, :tags]
   before_action :set_user_with_activities, only: [:activities]
   before_action :set_user_with_associations_and_statistics, only: [:show]
-  skip_before_action :authenticate_user, only: [:login, :authenticate]
+  skip_before_action :authenticate_user, only: [:authenticate]
```

Comment:
not sure you'd be needing this line anymore! since we're trying to abstract authentication